app/models/decision_review.rb

```diff
@@ -168,6 +169,11 @@ def contestable_issues
     contestable_issues_from_ratings + contestable_issues_from_decision_issues
   end
 
+  def active_nonrating_request_issues
+    @active_nonrating_request_issues ||= EndProductEstablishment.by_veteran_file_number(veteran.file_number)
```

Comment:
Should we add both end product active status & veteran file number to request issue to make this query not as big?

Reply:
Yeah i think we probably want this query made from a `RequestIssue` perspective. Since not all request issues will have an `EndProductEstablishment`, but we still want to select those.

```suggestion
    @active_nonrating_request_issues ||= RequestIssue.nonrating.select(&:status_active?)
```